dcicutils/misc_utils.py

```diff
@@ -49,6 +51,28 @@ def __call__(self, *args, timestamped=False, **kwargs):
 PRINT.__name__ = 'PRINT'
 
 
+absolute_uri_validator = (
+    rfc3986.validators.Validator()
+    # Validation qualifiers
+    .allow_schemes('http', 'https')
+    # TODO: We might want to consider the possibility of forbidding the use of a password. -kmp 20-Apr-2021
+    # .forbid_use_of_password()
+    .require_presence_of('scheme', 'host')
+    .check_validity_of('scheme', 'host', 'path'))
+
+
+def is_valid_absolute_uri(text):
+    try:
```

Comment:
Might be worth linking documentation/RFC on this. And also maybe providing the "backstory".

Reply:
Done.